### ISSUE TITLE
[BUG]: Superman Returns : Rendering anomalies once ingame.

### ISSUE THREAD

**Issue by gitpro03** [PCSX2/pcsx2#5963]
### Describe the Bug

When going in game in the game "Superman Returns", there are 2 main issues (if we put hardware rendering aside) which are subtle, but noticeable if you know where to look at these.

None of those problems can be fixed with a gamefix or any settings.

**1 - Minor SPS at some specific areas in the open world.**

For some reasons, the game is producing various SPS when ran into PCSX2 which are not present on original hardware. Yes, there is a gamefix to eliminate most of those, but there are still a couple of those artifacts from time to time in a couple of specific areas.

There is a savestate bundled with the blockdump, in the slot 0, which lead to one of those areas (you have to move back and forth to see the problem).

**2 - Missing geometry at the lower-left corner of the screen.**

That's the most weird problem of the bunch, for some very odd reasons, the game is not rendering some geometry at the lower-left corner of the screen. This does not happen on original hardware where none of the geometry shown on screen disapear in such a fachion.

As with the first problem, a savestate, in slot 1, will lead you to an area where the issue is very obvious (you will need to turn the camera, left or right, to see the problem)

### Reproduction Steps

Load the blockdump and select/load one of the 2 savestates bundled with the blockdump and observe the issues mentioned in the first section of the report.

### Expected Behavior

The game should not produce weird SPS in any areas, and should display visible geometry all the time if on-screen.

### PCSX2 Revision

pcsx2-v1.7.2656-windows-x86

### Operating System

Windows 11

### CPU

Intel core I9 9900K

### GPU

Nvidia Gefore RTX 3070

### GS Settings

The software render is required, the game uses the "VIS Clear".

### GS Window Screenshots

This is the first bug mentioned in the topic, minor SPS in some specific areas, this is how you should test the game to trigger the SPS issues:

https://user-images.githubusercontent.com/104388474/165183484-1bfb6099-e27a-49c9-bd83-747784126a0d.mp4

And this is the second problem, disappearing geometry at the lower-left portion of the screen:

First, a comparison between PCSX2 and real hardware:

This is PCSX2 (pay attention to the red circle):

![Capture d’écran 2022-04-25 235411](https://user-images.githubusercontent.com/104388474/165183764-ba722ea7-0b61-4ec3-83d3-6c2c0cc7d1c2.jpg)

And real hardware, see the difference:

![Capture d’écran 2022-04-25 235531](https://user-images.githubusercontent.com/104388474/165183807-78304633-e1de-4b55-8da0-2f0be18638a3.png)

Now the bug in motion:

https://user-images.githubusercontent.com/104388474/165183841-42a572d8-fa49-4f12-8659-d4af895fa4a5.mp4

### Logs & Dumps

This is the Blockdump + savestates package for testing:

https://drive.google.com/file/d/1RynxtAWa4jG-0KxQ2oTUOWYMe5f-HBMx/view?usp=sharing

MD5 of the iso file: 8c502adc891b80affe269115b9237370 (matches the redump entry so the iso file is correct - http://redump.org/disc/26391/).

**Comment by Mrlinkwii** [comment]
can this be check on latest nightly :)

**Comment by Technostrophe** [comment]
Still an issue as of v1.7.4075

**Comment by Technostrophe** [comment]
A year later, remains an issue as of 1.7.5590

**Comment by LoStraniero91** [comment]
The diseapparing geometry is actually a side-effect of using the VU Overflow Hack, but that's needed to fix the major SPS issues.

**Comment by refractionpcsx2** [comment]
yeah the overflow hack doesn't really fix much, we can't magically use overflows properly, but yeah it's enough to stop the game exploding.

**Comment by Technostrophe** [comment]
The first bug is a thing of the past, tried for ten minutes to trigger it in various areas of the City, the second bug is still in full effect.
